packages/cli/test/commands/deploy.test.ts

```diff
@@ -41,6 +41,29 @@ describe('deploy', () => {
       })
     })
 
+    fancy.stdout().it('calls pruneDependencies function', async (ctx) => {
+      const fakeProvider = {} as ProviderLibrary
+
+      const fakeLoader = fake.resolves({
+        provider: fakeProvider,
+        appName: 'fake app',
+        region: 'tunte',
+        entities: {},
+      })
+
+      const fakeDeployer = fake((_config: unknown, logger: Logger) => {
+        logger.info('this is a progress update')
+      })
+
+      const pruneDependenciesSpy = spy(deploy, deploy.pruneDependencies)
+      await runTasks('test-env', fakeLoader, fakeDeployer)
+        
+      expect(pruneDependenciesSpy).to.have.been.calledOnce
```

Comment:
Maybe this should be
```suggestion
      expect(deploy.pruneDependencies).to.have.been.calledOnce
```